src/code_checker.bzl

```diff
@@ -18,11 +18,21 @@ COMPILE_COMMANDS_JSON=$1
 shift
 COMPILE_COMMANDS_ABS=$COMPILE_COMMANDS_JSON.abs
 sed 's|"directory":"."|"directory":"'$(pwd)'"|g' $COMPILE_COMMANDS_JSON > $COMPILE_COMMANDS_ABS
-echo "Running: $@" $COMPILE_COMMANDS_ABS > $LOG_FILE
-echo "==================================" >> $LOG_FILE
+echo "CodeChecker command: $@" $COMPILE_COMMANDS_ABS > $LOG_FILE
+echo "===-----------------------------------------------------===" >> $LOG_FILE
+echo "                   CodeChecker error log                   " >> $LOG_FILE
+echo "===-----------------------------------------------------===" >> $LOG_FILE
 eval "$@" $COMPILE_COMMANDS_ABS >> $LOG_FILE 2>&1
 # ls -la $DATA_DIR
 # NOTE: the following we do to get rid of md5 hash in plist file names
+ret_code=$?
+echo "===-----------------------------------------------------===" >> $LOG_FILE
+if [ $ret_code -eq 1 ]; then
```

Comment:
Just wonder if we should check for other non zero codes?

Reply:
According to the CodeChecker docs (also, final few lines of `CodeChecker analyze --help`:
```
Exit status
------------------------------------------------
0 - Successful analysis
1 - CodeChecker error
3 - Analysis of at least one translation unit failed
128+signum - Terminating on a fatal signal whose number is signum
```
So it would make sense to check for >128.